ext/ffi/ffi.c

```diff
@@ -2968,6 +2968,30 @@ static zend_always_inline bool zend_ffi_validate_api_restriction(zend_execute_da
 		} \
 	} while (0)
 
+#ifdef PHP_WIN32
+# define NUM_MODULES 1024
+/* A rough approximation of dlysm(RTLD_DEFAULT) */
+static void *dlsym_loaded(char *symbol)
+{
+	HMODULE modules[NUM_MODULES];
+	DWORD num, i;
+	void * addr;
+	if (!EnumProcessModules(GetCurrentProcess(), modules, sizeof modules, &num)) {
+		return NULL;
+	}
+	if (num >= NUM_MODULES) {
+		num = NUM_MODULES - 1;
```

Comment:
It's better to allocate a bigger  temporary buffer repeat ``EnumProcessModules()`` call and process all the modules.

Reply:
Done.